R/cal-apply.R

```diff
@@ -0,0 +1,98 @@
+#---------------------------------- Apply --------------------------------------
+
+#' Applies a calibration to a set of prediction probabilities
+#' @details It currently supports data.frames only. It extracts the `truth` and
+#' the estimate columns names, and levels, from the calibration object.
+#' @param .data An object that can process a calibration object.
+#' @param calibration The calibration object (`cal_object`).
+#' @param ... Optional arguments; currently unused.
```

Comment:
Do we check anywhere that there are empty `...`? Otherwise, a misspelled argument will silently be passed on and the error might be non-sensical.